src/statsmodels.jl

```diff
@@ -0,0 +1,549 @@
+
+
+### Util functions
+
+StatsBase.responsename(m) = coefnames(formula(m).lhs)
+
+function baselevel(ct::CategoricalTerm)
```

Comment:
Can you prefix `CategoricalTerm` with the package that defines the type for clarity? Also for the below method definitions.